dev/archery/archery/docker/cli.py

```diff
@@ -216,7 +216,7 @@ def docker_run(obj, image, command, *, env, user, force_pull, force_build,
     PYTHON=3.8 archery docker run conda-python
 
     # disable the cache only for the leaf image
-    PANDAS=master archery docker run --no-leaf-cache conda-python-pandas
+    PANDAS=main archery docker run --no-leaf-cache conda-python-pandas
```

Comment:
can you also update this to `upstream_devel`?

Reply:
Yes, I have updated this sample command too.